docs/admin/uninstall.md

```diff
@@ -4,10 +4,10 @@ Here you will find any steps necessary to cleanly remove the App from your Nauto
 
 ## Database Cleanup
 
-Prior to removing the plugin from the `nautobot_config.py`, run the following command to roll back any migration specific to this plugin.
+Prior to removing the app from the `nautobot_config.py`, run the following command to roll back any migration specific to this app.
 
 ```shell
-nautobot-server migrate nautobot_plugin_secrets_providers zero
+nautobot-server migrate nautobot_secrets_providers zero
```

Comment:
Just trying to make sure if the command is supposed to be this.

```suggestion
nautobot-server migrate nautobot_app_secrets_providers zero
```

Reply:
The previous command was not valid and came from an issue with the old cookie.